<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 41, 47);</string>
  </property>
  <widget class="QLabel" name="Choose_your_country_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>355</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Choose you country</string>
   </property>
  </widget>
  <widget class="QLabel" name="Agree_label">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>440</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>I confirm that I am over 13 years old, and agree to the user agreement.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>275</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(74, 79, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="LeftBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>20</width>
     <height>740</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Registration_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 rgb(0, 200, 250), 
				stop: 0.4 rgb(30, 150, 255),
                stop:1 rgb(60, 90, 255));
	border: 0px;
}
QPushButton:hover
{
	
	background-color: rgb(25, 206, 255);
}</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="iconSize">
    <size>
     <width>220</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>-4</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="UpperBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Close_button">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{	
	background-color: rgba(0, 0, 0, 0);
	border: 0px; 
}
QPushButton:hover
{ 
	background-color: rgb(249, 22, 34);
 }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-DesktopClient_qt6-Desktop_Qt_6_6_0_MSVC2019_64bit-Debug/ApplicationIcons/CloseCross.png</normaloff>../build-DesktopClient_qt6-Desktop_Qt_6_6_0_MSVC2019_64bit-Debug/ApplicationIcons/CloseCross.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="GoBack_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgba(255, 255, 255, 0);
	border: 0 px;
	color: rgb(182, 182, 182);
}
QPushButton:hover
{		
	color: rgb(220, 220, 220);
}</string>
    </property>
    <property name="text">
     <string>&lt; Go back</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Agree_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>441</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>0</number>
   </property>
   <property name="autoRepeatInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="Email_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>255</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(182, 182, 182);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(74, 79, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="Countries_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>375</y>
     <width>600</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border: 0px;
background-color: rgb(74, 79, 90);</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>USA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Canada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Russia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>France</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Germany</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Japan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>China</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Create_account_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </widget>
  <widget class="QFrame" name="Right_Border_frame">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>20</width>
     <height>740</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="BottomBorder_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>770</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 66, 75);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Error_window_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>600</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(255, 50, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
